src/hist1d.jl

```diff
@@ -3,7 +3,7 @@ struct Hist1D{T<:Real,E} <: AbstractHistogram{T,1,E}
     sumw2::Vector{Float64}
     hlock::SpinLock
     # most concrete inner constructor
-    function Hist1D(h::Histogram{T,1,E}, sw2 = zeros(Float64, size(h.weights))) where {T,E}
+    function Hist1D(h::Histogram{T,1,E}, sw2 = h.weights) where {T,E}
```

Comment:
h.weights .^2 or no

Reply:
originally I had h.weights.^2 but then the math is screwed up. This default assumes w=1 for every event. In the case where we compute a statsbase Histogram with weighted entries, we also compute it for w^2 and override `sw2`

[`a88a351` (#10)](https://github.com/Moelf/FHist.jl/pull/10/commits/a88a3510d41716c9d99da6d4797556edaa91d3d3#diff-f405f95fa7246dd9a787e894c2107f6aed87f4f673a985b89b25f1628b1070a3R148-R150)

Reply:
ah yes, if weights were unit originally, this is fine. If weights were not, then doing ^2 here isn't correct either.